front/src/applications/editor/tools/trackEdition/utils.ts

```diff
@@ -12,7 +12,7 @@ import type { LinearMetadataItem } from 'common/IntervalsDataViz/types';
 
 import type { TrackEditionState, TrackSectionEntity } from './types';
 
-function getNewLine(points: [number, number][]): TrackSectionEntity {
+export function getNewLine(points: [number, number][]): TrackSectionEntity {
```

Comment:
```suggestion
function getNewLine(points: [number, number][]): TrackSectionEntity {
```

Reply:
I need this function to build the state of the split tool , so I don't understand this change.
https://github.com/OpenRailAssociation/osrd/blob/40fedcb19ff0d6fa6d7ef20434cf193eb530019a/front/src/applications/editor/tools/trackSplit/tool.tsx#L22

I can put this function in the `editor/tools/utils.ts` because now it is used in several tools